### ISSUE TITLE
Canon 70d

### ISSUE THREAD

**Issue by maverickvergara** [Doug-Pardee/LightZombie#17]
I just got a Canon 70d and love using lightzone, but it is very slow now and I would assume it is because there isn't a raw tone curve for it, but then again I am very new to this. Is there a way to get that?
Thanks for all your work,
Vic Vergara

**Comment by texandrews** [comment]
On 9/23/2013 3:05 PM, maverickvergara wrote:

> I just got a Canon 70d and love using lightzone, but it is very slow 
> now and I would assume it is because there isn't a raw tone curve for 
> it, but then again I am very new to this. Is there a way to get that?
> Thanks for all your work,
> Vic Vergara
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/Doug-Pardee/LightZombie/issues/17.
> 
> Is the 70D the brand new one?  Canon has always confused me with their 
> names and numbers.  it may be that the currentv4.0 doesn't have it, but 
> we are about to go to beta with a new v4.1, and that includes new raw 
> curves---maybe this is one---plus other stuff (which is why we have to 
> beta it).  In any event, once there is a dcraw profile plus some image 
> files available, we can profile it. See Doug's video  & etc. in the raw 
> profiling center in the block in the left sidebar.  Plus, looks like 
> we're going to have a team working on raw profiles now, not just Doug.

tex

PS:  watch that video and you can see how to roll your own.  Not sure if 
any other software is giving users that capability!

**Comment by maverickvergara** [comment]
Yes the 70d is the one that came out last month which is why I figured that was the case. I really appreciate your quick feedback. I have only been using Lightzone for a few months and really the first editing software I have really used other than watching other photographers use various programs, but I love what Lightzone offers.
Again thanks for the input,
Vic

Date: Mon, 23 Sep 2013 12:10:48 -0700

**Comment by Doug-Pardee** [comment]
The 70D isn't yet supported by dcraw. I have no idea when it will be, but it'll certainly be at least a month. It could be many months. DNG Converter 8.2, which adds support for the 70D, came out less than a week ago. Once dcraw does come out with 70D support, we have to make the LightZone version of that dcraw.

The slowness that you're seeing is in the Browser. The Canon 6D, SL1/100D, and apparently the 70D use uncompressed JPEG preview images in the Raw file. The graphics library that LightZone uses doesn't support uncompressed JPEGs. This is a known problem, and one that we're trying to figure out how to remedy.
